src/convertToObject.js

```diff
@@ -13,6 +13,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const sourceArray = sourceString.split(';');
+  const property = [];
+
+  for (let i = 0; i < sourceArray.length; i++) {
+    property.push(sourceArray[i].split(':'));
+  }
+
+  const styles = {};
+
+  for (let i = 0; i < property.length; i++) {
+    if (property[i][1] !== undefined) {
+      styles[property[i][0].trim()]
+        = property[i][1].trim();
+    }
```

Comment:
0, 1, i, если не я писал этот код, сложно разобраться
создай переменные для properties[i], properties[i][0], properties[i][1]

```suggestion
    if (property[i][1] !== undefined) {
      styles[property[i][0].trim()] = property[i][1].trim();
    }
```